.github/workflows/govulcheck.yaml

```
@@ -0,0 +1,22 @@
+name: Run govulncheck
+on:
+   schedule:
+    - cron: "0 12 * * *" # 8am EST / 10am PST / 12pm UTC
+   workflow_dispatch: # allow manual trigger for testing
```

This will end up triggering the workflow on this PR for testing too
```suggestion
  schedule:
    - cron: "0 12 * * *" # 8am EST / 10am PST / 12pm UTC
  workflow_dispatch: # allow manual trigger for testing
  pull_request:
    paths:
      - ".github/workflows/govulncheck.yml"
```

---

thanks i'll add that!